### ISSUE TITLE
Некорректная работа проверки "doc-comment-export-function-return-section" совместно с устаревшим методом (#std453.5.7)

### ISSUE THREAD

**Issue by unichkin** [1C-Company/1c-edt-issues#707]
### Описание ошибки

Если я перемещаю метод в другой модуль, старое место вызова оставляя как заглушку (с примечанием "Устарела") - проверка ругается на отсутствие параметров. Но их по нотации переноса надо брать уже из нового модуля.

https://its.1c.ru/db/v8std#content:453:hdoc

### Как воспроизвести

1) Создать два модуля "Новый" и "Старый", в обоих добавить экспортную функцию "Тест"
2) Описать ее:

**Старый:**
// Устарела. Следует использовать см. Новый.Тест
Функция Тест(Парам1, Парам2) Экспорт
КонецФункции

**Новый**
// Тест
// Параметры:
//   Парам1 - Произвольный
//   Парам2 - Произвольный
// ВозвращаемоеЗначение:
//   Произвольный
// 
Функция Тест(Парам1, Парам2) Экспорт
   Возврат Неопределено;
КонецФункции

### Ожидаемое поведение

Подсказка должна опознавать такие ситуации, и не ругаться на отсутствие параметров в старом устаревшем методе. 

### Лог рабочей области

Лог не требуется, описания достаточно

### Версия 1С:EDT

Ruby 2022.1.2

### Операционная система

Windows

**Comment by RedMammoth** [comment]
А мы разве должны убирать описание в старой процедуры. Она ведь не удаляется для сохранения совместимости и у нее может отличаться количество/типы параметров от новой. Может правильней просто добавлять первой строкой примечание, что функция устарела со ссылкой на новую:

```
// Устарела. Следует использовать См. НовыйМодуль.МояФункция
// 
// Параметры:
//  Парам1 - Строка - Парам1
//  Парам2 - Произвольный - Парам2
// 
// Возвращаемое значение:
//  Строка - Моя функция
//
Функция МояФункция(Парам1, Парам2) Экспорт
	Возврат "Тест";
КонецФункции

```

**Comment by unichkin** [comment]
Да, еще момент. По стандарту такие методы должны помещаться в область "УстаревшиеПроцедурыИФункции", такой проверки сейчас нет.

**Comment by RedMammoth** [comment]
В работе https://github.com/1C-Company/v8-code-style/pull/1136

**Comment by marmyshev** [comment]
[Как ссылаться на типы метода](https://github.com/1C-Company/v8-code-style/blob/master/docs/checks/code_typification.md#наследование-типов-параметров-по-сигнатуре-метода)
